impedance/models/circuits/circuits.py

```diff
@@ -188,7 +188,7 @@ def __str__(self):
             for name, value in self.constants.items():
                 elem = get_element_from_name(name)
                 units = check_and_eval(elem).units
-                if '_' in name:
+                if '_' in name and len(units)>1:
```

Comment:
Needs whitespace around the operator

```suggestion
                if '_' in name and len(units) > 1 :
```